sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/implementation/UserAgentContainerTest.java

```diff
@@ -87,7 +87,7 @@ private String getUserAgentFixedPart() {
         osName = osName.replaceAll("\\s", "");
```

Comment:
btw, this class, has a test named `UserAgentIntegrationTest()` (from prior useragent PR), please change that to
`userAgentIntegration()` to follow test name code style.